homeassistant/components/sharkiq/config_flow.py

```diff
@@ -116,9 +116,15 @@ async def async_step_user(
         )
 
     async def async_step_reauth(
-        self, user_input: Mapping[str, Any]
+        self, entry_data: Mapping[str, Any]
     ) -> ConfigFlowResult:
         """Handle re-auth if login is invalid."""
+        return await self.async_step_reauth_confirm()
```

Comment:
Small change in behavior...
Previously, the original credentials were tested, now the user is prompted for new credentials without attempting the old ones.
The logic with `if user_input is not None:` implies that this was the desired behavior.